src/webgpu/api/operation/device/info.spec.ts

```diff
@@ -0,0 +1,34 @@
+export const description = `
+Tests GPUDevice.adapterInfo member.
+`;
+
+import { Fixture } from '../../../../common/framework/fixture.js';
+import { makeTestGroup } from '../../../../common/framework/test_group.js';
+import { getGPU } from '../../../../common/util/navigator_gpu.js';
+import { assert, objectEquals } from '../../../../common/util/util.js';
+
+export const g = makeTestGroup(Fixture);
+
+g.test('device_adapter_info')
+  .desc(
+    `
+  Test that GPUDevice.adapterInfo matches GPUAdapter.info`
+  )
+  .fn(async t => {
+    const gpu = getGPU(t.rec);
+    const adapter = await gpu.requestAdapter();
+    assert(adapter !== null);
+
+    const device = await t.requestDeviceTracked(adapter);
+    assert(device !== null);
+
+    assert(device.adapterInfo instanceof GPUAdapterInfo);
+    assert(adapter.info instanceof GPUAdapterInfo);
+
```

Comment:
Should test the `[SameObject]` property, i.e.:

`t.expect(device.adapterInfo === device.adapterInfo);`

This should be tested for `adapter.info` too if it is not already.

Reply:
This can already be tested at least for adapter.info, as the spec already says `[SameObject]`.

Reply:
You're right!